VoltAir/qml/actors/Orb.qml

```diff
@@ -59,16 +58,24 @@ Actor {
         cacheRenderParams: true
         cullingControlsBodyActive: true
         pauseWhenHidden: true
-        frameCount: 11
+        frameCount: 1 //11
```

Comment:
If there will be an animated background. Zap object has 11 frames, but if something else is used, it might have a different number of frames.